src/components/RecycleScroller.vue

```diff
@@ -201,6 +201,15 @@ export default {
     })
   },
 
+  activated() {
+    const lastPosition = this.$_lastUpdateScrollPosition;
+    if (typeof lastPosition === 'number') {
+      this.$nextTick(() => {
+        this.scrollToPosition(lastPosition);
```

Comment:
Does this have to be nextTick? The DOM should be ready by the time activated is called.

Reply:
It has been a long time. i do not remember very clearly. May be it is because when (positionDiff < itemSize) in updateVisibleItems method, something goes wrong with view calculated, So i make it asynchronous